listings/ch00-introduction/constructor/src/lib.cairo

```diff
@@ -0,0 +1 @@
+mod constructor
```

Comment:
Can you directly write the contract in the `lib.cairo` file?
Also, here you're missing a `;` after your `mod constructor` line.